testplan/testing/multitest/suite.py

```diff
@@ -357,6 +357,7 @@ def _testcase(func):
 
 
 def _testcase_meta(
+    name=None,
```

Comment:
I think it's confusing to have a name param in @testcase decorator. Intuitively the name of a testcase is the function name. We can have a display_name for the testcase though. I suggest we change name ==> display_name here.

Reply:
the whole point of this PR is to provide a way to override the name of a testcase via the decorator. The function name is just the default string used for the name, it has no other special meaning.